doc/en/user/source/community/features-templating/configuration.rst

```diff
@@ -1,610 +1,102 @@
+
+The :guilabel:`Workspace` and :guilabel:`Layer Name` values, if specified, will affect also the place where the template will be saved:
+
+* if none is specified the template will be saved inside the :code:`features-templating` directory.
+* if a :guilabel:`Workspace` is specified the template will be saved in that workspace folder.
+* if a :guilabel:`Layer Name` is specified the template will be saved in that layer folder.
+
+
+The :guilabel:`Template Content` section is where the template is actually defined. 
+
+* The template can be uploaded from a file, and in that case the :guilabel:`Template Name` and :guilabel:`Template File Type` fields are automatically populated from the file.
+* Otherwise the template can be written from scratch into the template editor.
+
+By clicking on the :guilabel:`Preview` tab the user can specify parameters to test the template and preview the result. The preview will return only a single feature.
+
+.. warning:: When previewing template the template gets saved/updated in the data directory. This is due the fact that the preview works by issuing a WFS request. This implies that the previous state is lost, but also that any modification is immediately visible to a user that might be accessing the layer.
+
+.. figure:: images/preview-ui.png
+
+* The user must specify one value among the :guilabel:`Available Output Formats`
+* The user must specify values among the available for fields :guilabel:`Workspace` and :guilabel:`Layer Name`.
+* If the user specified a  :guilabel:`Workspace` for the template in the :guilabel:`Data` tab the preview :guilabel:`Workspace` will be automatically set from that one.
+* If the user specified a  :guilabel:`Layer Name` for the template in the :guilabel:`Data` tab the preview :guilabel:`Layer Name` will be automatically set from that one.
+* The user can specify a :guilabel:`Feature ID` to obtain a preview for the specified feature.
+* The user can specify a :guilabel:`CQL Filter` to obtain a preview for a feature matching the filter.
+
+
+The :guilabel:`Validate` button acts differently according to the output format:
+
+* In the GML case will trigger a schema validation based on the Schema Location specified in the template.
+
+* In the JSON-LD case will perform a JSON-LD ``@context`` validation.
+
+* In the GeoJSON case no validation will occur.
+
+
+Add  Templates Rules to a Layer
+--------------------------------
+
+To inform GeoServer when to apply a template, the user needs to specify rule on a per layer basis. 
+The most basic rule is one that bounds a template to specific output format. :guilabel:``Request CQL Functions`` allow to specify more advanced rules.
```

Comment:
Not sure I follow here. Are you trying to say "binds the template to the format", or "limits the template to the format"? I'm assuming the first with this suggestion:
```suggestion
The most basic rule is one that binds a template to a specific output format. :guilabel:``Request CQL Functions`` allow specifying more advanced rules.
```